banzai/dbs.py

```diff
@@ -82,11 +82,12 @@ class Instrument(Base):
     __tablename__ = 'instruments'
     id = Column(Integer, primary_key=True, autoincrement=True)
     site = Column(String(15), ForeignKey('sites.id'), index=True)
```

Comment:
Shouldn't site also be nullable=False, or is that done automatically by virtue of being declared a foreign key?